README.md

```diff
@@ -973,6 +972,61 @@ Learn more about class-transformer and how to handle more complex object constru
 This behaviour is enabled by default.
 If you want to disable it simply pass `useClassTransformer: false` to createExpressServer method.
 
+## Auto validating action params
+
+Sometimes parsing a json object into instance of some class is not enough. 
+E.g. `class-transformer` doesn't check whether the property's types are correct, so you can get runtime error if you rely on TypeScript type safe. Also you may want to validate the object to check e.g. whether the password string is long enough or entered e-mail is correct.
+
+It can be done easily thanks to integration with [class-transformer-validator][7] which internally depends on [class-validator][6]. So first of all, you have to install the modules:

```

Comment:
> not your npm module

I don't care about it, this package is for open-source, not for me. I do care about complexity and maintainability. I have lot of packages and know about maintainability costs when you have lot of packages to maintain, and this small extra package to maintain does not cost it. What exactly this package does? How can you use it without routing-controllers? Isn't it tightly bind to routing-controllers? Or is there anything non-specific to routing-controllers? Maybe routing-controllers does not need it at all? And just have only core really needed from this package inside routing-controllers, and you can use your package with extra specific features you need?